cupyx/scipy/sparse/sputils.py

```diff
@@ -1,5 +1,13 @@
 import cupy
 
+from cupy.core._dtype import get_dtype
+
+supported_dtypes = ['single', 'double', 'csingle', 'cdouble']
+supported_dtypes = [get_dtype(x) for x in supported_dtypes]
```

Comment:
No intermediate variable is necessary.
```suggestion
supported_dtypes = [get_dtype(x) for x in ('single', 'double', 'csingle', 'cdouble')]
```